src/store/models/Notification.js

```diff
@@ -56,10 +57,13 @@ export function urlForNotification ({ activity: { action, post, comment, group,
       return postUrl(post.id, { groupSlug })
     case ACTION_NEW_COMMENT:
       return postCommentUrl({ postId: post.id, commentId: comment.id, groupSlug })
-    case ACTION_TAG:
+    case ACTION_TAG: {
       const tagReason = find(r => r.startsWith('tag: '), reasons)
       const topicName = tagReason.split(': ')[1]
       return postUrl(post.id, { groupSlug, topicName })
+    }
+    case ACTION_NEW_POST:
+      return postUrl(post.id, { groupSlug })
```

Comment:
This reminds me that it would be good to have a different path when the post is a chat-post, so it goes straight to the chat room. I thought I worked on something like that but might just have been chat paths from node repo

Reply:
you are right, I will do this

Reply:
oh actually this already works, if there is a topic on the post it goes to ACTIOB_TAG above which goes to the chat room with the post open